lib/spack/spack/build_environment.py

```diff
@@ -253,6 +258,7 @@ def set_module_variables_for_package(pkg, module):
     # TODO: Currently, everything is a link dependency, but tools like
     # TODO: this shouldn't be.
     m.cmake = Executable('cmake')
+    m.ctest = Executable('ctest')
```

Comment:
ctest is part of the cmake package. It behaves in many of the same ways, including accepting a `-j` option to run the tests in parallel. No packages use it right now, but Psi4 uses it and I don't feel like editing this file in multiple PRs until the flake8 fixes are merged.